tests/python/test_linalg.py

```diff
@@ -179,14 +182,12 @@ def fill():
 def test_any_all():
     a = ti.Matrix(2, 2, dt=ti.i32, shape=())
     b = ti.var(dt=ti.i32, shape=())
+    c = ti.var(dt=ti.i32, shape=())
```

Comment:
Does this test need to be changed?

Reply:
No, just an off-topic-trvial changes, see ctrb_guide.